### ISSUE TITLE
Decklists break anchor links

### ISSUE THREAD

**Issue by fenhl** [NilsEnevoldsen/ScryfallLinks#29]
The `<d>` decklists make anchor links point at the wrong section of the page, because they change the height of the page after it finishes loading. For an example, follow the deck links at <https://mtg.gamepedia.com/Magic:_The_Gathering_Arena#Starter_pack>.

**Comment by NilsEnevoldsen** [comment]
I was unable to replicate this with either Safari or with Chrome throttled to "Slow 3G" network conditions with developer tools.

What browser are you using? How fast is your connection?

**Comment by fenhl** [comment]
Firefox on a ping of 40ms and a download of 10Mb/s.

**Comment by NilsEnevoldsen** [comment]
Yes, I can replicate this on Firefox. I don't even need to clear the cache. Thanks.

I don't know what to do about this, so suggestions are welcome!

**Comment by NilsEnevoldsen** [comment]
@fenhl I can no longer replicate on Firefox. Can you?
